apps/rush-lib/src/api/RushConfiguration.ts

```diff
@@ -908,7 +908,7 @@ export class RushConfiguration {
     this._yarnCacheFolder = path.resolve(path.join(this._commonTempFolder, 'yarn-cache'));
 
     this._changesFolder = path.join(this._commonFolder, RushConstants.changeFilesFolderName);
-    this._rushUserFolder = path.join(Utilities.getHomeDirectory(), '.rush');
+    this._rushUserFolder = path.join(Utilities.getHomeDirectory(), '.rush', `node-${process.version}`);
```

Comment:
Can patch releases be considered equivalent?

Reply:
I think it may make sense to add a command to delete other versions of node. That can be added in another PR, though.

---
In reply to: [231271442](https://github.com/Microsoft/web-build-tools/pull/934#discussion_r231271442) [](ancestors = 231271442)